common/static/js/vendor/image-zoom-tool.js

```diff
@@ -0,0 +1,334 @@
+/*
+ * edX Image Zoom Tool
+ * v0.0.1
+ *
+ * This script was modified from our jquery.loupeAndLightbox.js script, but 
+ * goes further and adds keyboard accessibility to the zooming controls.
+*/
+
+(function($){
+    $.fn.edxImageZoomTool = function(options) {
+        var settings = $.extend({}, $.fn.edxImageZoomTool.defaults, options),
+            keys = {
```

Comment:
This script is uploaded through Studio. Do I have access to HtmlUtils.constants? If so, I'll use those keys instead of these defined keys.